Source/MQTTnet/Client/Options/MqttClientTlsOptions.cs

```diff
@@ -29,6 +29,8 @@ public sealed class MqttClientTlsOptions
 
 #if NETCOREAPP3_1 || NET5_0_OR_GREATER
         public List<System.Net.Security.SslApplicationProtocol> ApplicationProtocols { get; set; }
+        
+        public System.Net.Security.CipherSuitesPolicy CipherSuitesPolicy { get; set; }
```

Comment:
Is _null_ a proper value for this property? I do not know details about the new API in .net...

Reply:
Yes, it is the default value.
See https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslserverauthenticationoptions.ciphersuitespolicy?view=net-6.0#system-net-security-sslserverauthenticationoptions-ciphersuitespolicy

> When set to null, the operating system default is used.